t/client-monitoreditems-multiple.t

```diff
@@ -160,12 +159,18 @@ is($client->{client}->MonitoredItems_deleteSingle($subid, $monids[0]),
    "Good",
    "delete monitored item");
 
-my $items;
+my @items;
 for (0 .. 2) {
-    push @$items, dclone $request->{CreateMonitoredItemsRequest_itemsToCreate}[0];
-    $items->[$_]{MonitoredItemCreateRequest_itemToMonitor}{ReadValueId_nodeId} = $nodes[$_];
+    push @items,
+	OPCUA::Open62541::Client->MonitoredItemCreateRequest_default({
```

Comment:
Now we can just pass "$nodes[$_]" here as argument to MonitoredItemCreateRequest_default() instead of calling it and overwriting the nodeId value.